test-integration/helpers/ui/user.coffee

```diff
@@ -104,8 +104,7 @@ class User extends TestHelper
       hider = document.createElement('style')
       hider.textContent = '''
         .navbar-fixed-bottom, .navbar-fixed-top,
-        .pinned-on .pinned-header,
-        .pinned-footer {
+        .pinned-on .pinned-header {
```

Comment:
Why was `.pinned-footer` removed? I just included all the selectors where `position: fixed` was set (rather than trying to decide which ones are ok to remain pinned and which were not ok)